### PR TITLE
Pimoroni tinyfx

### PR DESCRIPTION
**Pull request by bablokb**
Files for the newly added board (Pimoroni Tiny FX).
See https://github.com/adafruit/circuitpython/pull/9575

@ZodiusInfuser: FYI

### CODE REVIEW

File: _board/pimoroni_tinyfx.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: "Pimoroni"
+board_url:
+ - "https://shop.pimoroni.com/products/tinyfx"
+board_image: "pimoroni_tinyfx.jpg"
+date_added: 2021-09-02
+family: raspberrypi
+
+features:
+  - USB-C
+  - STEMMA QT/QWIIC
```
Comment (ZodiusInfuser) [inline _board/pimoroni_tinyfx.md:15]: Could this have `Speaker` listed? Or is that only for boards with an attached speaker, not just a speaker output?

File: _board/pimoroni_tinyfx.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+layout: download
+board_id: "pimoroni_tinyfx"
+title: "Tiny FX Download"
+name: "Tiny FX"
+manufacturer: "Pimoroni"
+board_url:
+ - "https://shop.pimoroni.com/products/tinyfx"
+board_image: "pimoroni_tinyfx.jpg"
+date_added: 2021-09-02
```
Comment (ZodiusInfuser) [inline _board/pimoroni_tinyfx.md:10]: Date needs updating

**Comment by ZodiusInfuser** [comment]
Thanks for this!

**Review by makermelissa — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**